profile/src/main/java/com/scottlogic/deg/profile/reader/AtomicConstraintReader.java

```diff
@@ -106,6 +140,9 @@ public Constraint readAtomicConstraintDto(AtomicConstraintDTO dto, ProfileFields
     private Object readAnyType(Field field, Object value) {
         switch (field.getType()) {
             case DATETIME:
+                if (!(value instanceof String)) {
+                    throw new ValidationException("Field " + field.name + " is a datetime but you are trying to assign non datetime value");
+                }
```

Comment:
Should this validation happen within `HelixDateTime`?  I thought the main reason for that class was to handle validation.